src/DeferredFutures.jl

```diff
@@ -47,6 +47,21 @@ function Base.show(io::IO, ref::DeferredFuture)
     print(io, "$(typeof(ref).name.name) at ($(rc.where),$(rc.whence),$(rc.id))")
 end
 
+"""
+    serialize(s::AbstractSerializer, ref::DeferredFuture)
+
+Serialize a DeferredFuture such that it can de deserialized by `deserialize` in a cluster.
+"""
+function Base.serialize(s::AbstractSerializer, ref::DeferredFuture)
+    if VERSION < v"0.6.0-rc1"
```

Comment:
If github says the commit came from a PR, click through to that PR, go down to where it was merged, then click on the commit link for the merge event. If it was a squash merge it'll be the same, but otherwise not.